ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -144,6 +144,16 @@ public Account getAccountById(short id) {
     return accountInfoMapRef.get().getAccounts();
   }
 
+  @Override
+  public short getContainerIdForLegacyPutPublicBlob() {
```

Comment:
Also "Legacy" is too vague a term - its definition changes. Let's give this a rethink - these really shouldn't be function IMO